flow/layers/image_filter_layer.cc

```diff
@@ -14,19 +14,22 @@ ImageFilterLayer::ImageFilterLayer(sk_sp<SkImageFilter> filter)
 void ImageFilterLayer::Preroll(PrerollContext* context,
                                const SkMatrix& matrix) {
   TRACE_EVENT0("flutter", "ImageFilterLayer::Preroll");
-
   Layer::AutoPrerollSaveLayerState save =
       Layer::AutoPrerollSaveLayerState::Create(context);
 
   SkRect child_bounds = SkRect::MakeEmpty();
   PrerollChildren(context, matrix, &child_bounds);
-  if (filter_) {
-    const SkIRect filter_input_bounds = child_bounds.roundOut();
-    SkIRect filter_output_bounds =
-        filter_->filterBounds(filter_input_bounds, SkMatrix::I(),
-                              SkImageFilter::kForward_MapDirection);
-    child_bounds = SkRect::Make(filter_output_bounds);
+
+  if (!filter_) {
+    set_paint_bounds(child_bounds);
+    return;
```

Comment:
This avoids https://github.com/flutter/engine/pull/23751/files#diff-ed00eb358e01399f0680370efff68aa55b41d3a886d3a0d9c9c7078323660a37R55, which previously would get hit in the ImageFilterLayerTest.EmptyFilter test from flow_unittests

Reply:
(line 55 below here, 52 in the original source)